raising-jiras.md

```diff
@@ -0,0 +1,87 @@
+#Raising Jiras
```

Comment:
I personally like the simple version when raining a jira:
The WHAT and the WHY - to be outlined by any stakeholder. The WHAT should detail the requirements, idea, bug improvement in what ever way explains this best. 

The WHY should be the business case which will help with prioritisation, the forecasted value, the operational saving on time/cost etc.

I don't think the HOW should be detailed until the work is prioritised. When the work is prioritised the development team will the plan the HOW to build and HOW to test using there own expertise. 

I think there should be a clear distinction between:
1. Raising a jira
2. Definition of Ready

We should be careful we are not spending time planning work that is not going to be prioritised and therefore should make a clear distinction between the two points I raised above. Definition of Ready (DOR) is a tool used by Scrum Teams in order to make sure they have what they need to pull a ticket into a Sprint. The DOR should also be an evolving list and may vary dependant on what each Scrum Team need.